src/makeRobot.js

```diff
@@ -38,7 +38,95 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    x: 0,
+    y: 0,
+    name: name,
+    wheels: wheels,
+    version: version,
+
+    get info() {
+      return 'name: '
+      + `${this.name}, `
+      + 'chip version: '
+      + `${this.version}, `
+      + 'wheels: '
+      + `${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.x}, y=${this.y}`;
+    },
+
+    goForward(num) {
```

Comment:
```suggestion
    goForward(num = 1) {
```
by doing so you'll avoid `if (!num) ...`